galaxy/templates/job-db-migrations.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: batch/v1
+kind: Job
```

Comment:
Very timely because it's probably time we revisited migrations in general.

I guess this is relying on the idea that Galaxy won't start without the proper database version set. Seems fine, but I'm not quite sure I understand why this job would run sooner than the init container? Could it cause the remaining containers to unnecessarily crash loop?

Reply:
The job would get dispatched from the beginning, and would be waiting for Postgres to come up along with the Postgres init containers. Another option would be to combine the two init containers in one of the deployments

Reply:
I can't remember what the outcome of the discussion on this was?